RIRImageResizing/Classes/NSObject/RIRResizeImageOperationParameters.h

```diff
@@ -0,0 +1,43 @@
+//
+//  RIRResizeImageOperationParameters.h
+//  Pods
+//
+//  Created by Richard Reitzfeld on 11/21/16.
+//
+//
+
+#import <Foundation/Foundation.h>
+#import <UIKit/UIKit.h>
+
+
+
+
+
+typedef NS_ENUM(NSInteger, UIImage_RIRResizing_ResizeMode) {
+    UIImage_RIRResizing_ResizeMode_ScaleToFill,
+    UIImage_RIRResizing_ResizeMode_AspectFit,
+    UIImage_RIRResizing_ResizeMode_AspectFill,
```

Comment:
Can you lower case all the last set of word components on each of these types, e.g. `UIImage_RIRResizing_ResizeMode_ScaleToFill` -> `UIImage_RIRResizing_ResizeMode_scaleToFill`. I think you should do a project find replace so you catch all the little things.